src/scripts/background/index.js

```diff
@@ -163,55 +153,59 @@ class BackgroundScript {
       case ACTION_INIT_CONTENT_SCRIPT:
         this.onRecieveInitContentScript();
         break;
+      default:
+        logger.error(`Invalid action: ${message[KEY_ACTION]}`);
+        break;
     }
   }
 
   onBrowserActionClickListener(tab) {
-    logger.debug('onBrowserActionClicked');
     if (this.isDisableDoubleClick()) return this.onSingleClickEvent();
     if (!this.isWaitingDoubleClick()) return this.setDoubleClickTimer();
-    this.clearDoubleClickTimer();
-    this.onDoubleClickEvent();
+    return this.clearDoubleClickTimer().onDoubleClickEvent();
   }
 
   onTabActivatedListener(activeInfo) {
-    this._setFocusTab(activeInfo);
+    this.setFocusTab(activeInfo);
     this.onActivateChanged(EventType.TAB_CHANGED);
   }
 
   onWindowFocusChangedListener(windowId) {
-    this.setFocusTabFromActivateWindow(windowId).then(tab => {
-      this.onActivateChanged(EventType.WINDOW_CHANGED);
-    });
+    this.setFocusTabFromActivateWindow(windowId).then(tab => this
```

Comment:
__[ESLint]__ <sub>posted by Sider</sub>

Warning - 'tab' is defined but never used. (no-unused-vars)